xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -256,6 +289,23 @@ def _skip_target(*, target, deps, deps_attrs, transitive_infos):
         ),
     )
 
+def _create_envs_depset(*, ctx, target, id, automatic_target_info):
+    raw_run_env = target[RunEnvironmentInfo].environment if RunEnvironmentInfo in target else {}
```

Comment:
If the `hasattr` check works, we need to use it here as well, otherwise this fails for Bazel < 5.3.